src/plugins/visualizations/public/embeddable/visualize_embeddable.ts

```diff
@@ -63,6 +63,7 @@ export interface VisualizeInput extends EmbeddableInput {
   query?: Query;
   filters?: Filter[];
   timeRange?: TimeRange;
+  newVisType?: string;
```

Comment:
This doesn't seem to be used

Reply:
Nope it's not. That's leftover from code I removed in #https://github.com/elastic/kibana/pull/94139. I'll remove it from here as well.